assets/js/googlesitekit/notifications/components/layout/BannerNotification.js

```diff
@@ -46,6 +47,7 @@ export default function BannerNotification( {
 	learnMoreLink,
 	dismissButton,
 	ctaButton,
+	dismissOnCTAClick,
```

Comment:
@benbowler why this property has been added? I don't see it is being used in this PR.

Reply:
This was because I checked out this file from @zutigrm's commit e11e62d4feaecc379b263f2bf4821255ef7ad52e so that we were using the same component and this prop came along with his updates. I can see that work is now merged and I have conflicts to fix which I will do now.